api/client/stats.go

```diff
@@ -139,7 +139,7 @@ func (s *containerStats) Display(w io.Writer) error {
 //
 // This shows real-time information on CPU usage, memory usage, and network I/O.
 //
-// Usage: docker stats [OPTIONS] [CONTAINER...]
+// Usage: docker stats [OPTIONS] CONTAINER [CONTAINER...]
```

Comment:
This is incorrect, because `CONTAINER` is optional now; don't providing a `CONTAINER` will show stats for all containers